src_bicep/predeploy/main.parameters.json

```diff
@@ -0,0 +1,14 @@
+{
+    "identityName": "",
+    "galleryName": "",
+    "network": {
+        "name": "vnetBuilder",
+        "addressPrefix": "10.0.0.0/16",
+        "addressPrefixSubnet": "10.0.0.0/24"
```

Comment:
It seems the first IP range is vnetAddressPrefix, and the second is defaultSubnetAddressPrefix.